packages/ariakit/src/popover/__examples__/popover-flip/test-chrome.ts

```diff
@@ -0,0 +1,11 @@
+import { devices, expect, test } from "@playwright/test";
+
+test("popover flip", async ({ page }, testInfo) => {
+  testInfo.snapshotSuffix = "";
+  // Resize the viewport to ensure it overflows
+  await page.setViewportSize(devices["iPhone 8"].viewport);
+  await page.goto("/examples/popover-flip");
+  await expect(page.locator("role=dialog[name='Team meeting']")).toBeVisible();
+  // Expect the popover to be placed on the bottom
+  expect(await page.screenshot()).toMatchSnapshot();
+});
```

Comment:
Also, let's take the screenshots of the preview container instead of the whole page, so these tests won't fail whenever we update the site layout. I know the popover may not be in total, but its position is what matters.